scanapi/__init__.py

```diff
@@ -18,14 +18,22 @@
     type=click.Path(exists=True),
     default=SETTINGS["spec_path"],
 )
-@click.option("-d", "--docs-path", "docs_path", default=SETTINGS["docs_path"])
+@click.option("-d", "--docs-path", "docs_path")
```

Comment:
Removed `default` value because now it will depend on the chosen reporter.

Reply:
Note for the future 😂 : We need to remember to update README.md with the new commands.

Reply:
Yeah for sure. That will be part of this PR too.

Reply:
https://github.com/camilamaia/scanapi/pull/56/commits/e899f01bfadeafb60c4a8022db2112f81a1455a3